modules/atmosphere/rendering/renderableatmosphere.cpp

```diff
@@ -88,30 +88,30 @@ namespace {
 
     constexpr openspace::properties::Property::PropertyInfo OzoneHeightScaleInfo = {
         "OzoneLayerHeightScale",
-        "Ozone Scale Height (KM)",
-        "It is the vertical distance over which the density and pressure fall by a "
-        "constant factor.",
+        "Ozone Scale Height (Km)",
```

Comment:
```suggestion
        "Ozone Scale Height (km)",
```

From https://www.bipm.org/documents/20126/41483022/SI-Brochure-9-EN.pdf/2d2b50bf-f2b4-9661-f402-5f9d66e4b507#page=33

> They are printed in lower-case letters unless they are derived from a proper name, in which
case the first letter is a capital letter.